### PR TITLE
preparing for CRAN release

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
- use consistent (80-width) code formatting
- improve .Rd docs
- add tests for global settings

### CODE REVIEW

File: R/error-checks.R [hunk 1]
```diff
@@ -19,7 +19,6 @@
 #' df <- data.frame(x = c(1, 2, 3))
 #' isOfType(df, "data.frame")
 #' @export
-
 isOfType <- function(object, type, nullAllowed = FALSE) {
```
Comment (IndrajeetPatil) [inline R/error-checks.R:22]: As decided during the `DataCombined` PR, no empty line between function and its `roxygen` docs.

File: R/ospsuite.utils-env.R [hunk 2]
```diff
@@ -1,36 +1,56 @@
...
+# this will be used to retrieve information on the package at runtime
 ospsuiteUtilsEnv$packageName <- "ospsuite.utils"
 
+# name of the suite to which this package belongs
 ospsuiteUtilsEnv$suiteName <- "Open Systems Pharmacology"
 
-# Default values for the formatNumerics() helper function
-ospsuiteUtilsEnv$formatNumericsDigits <- 2
+# default values for the `formatNumerics()` helper function
+ospsuiteUtilsEnv$formatNumericsDigits <- 2L
```
Comment (IndrajeetPatil) [inline R/ospsuite.utils-env.R:19]: This is always going to be an integer.

File: R/ospsuite.utils-env.R [hunk 3]
```diff
@@ -1,36 +1,56 @@
...
+#' an error is thrown.
 #'
 #' @examples
+#' getOSPSuiteUtilsSetting("packageName")
 #' getOSPSuiteUtilsSetting("suiteName")
+#' getOSPSuiteUtilsSetting("formatNumericsDigits")
+#' @export
 getOSPSuiteUtilsSetting <- function(settingName) {
   if (!(any(names(ospsuiteUtilsEnv) == settingName))) {
-    stop(ospsuite.utils::messages$errorPackageSettingNotFound(settingName, ospsuiteUtilsEnv))
```
Comment (IndrajeetPatil) [inline R/ospsuite.utils-env.R:29]: No need to use namespace operator since `messages` is in package namespace already.

File: R/utilities.R [hunk 4]
```diff
@@ -1,61 +1,67 @@
...
 #'
 #' @param x,y If `x` is `NULL`, will return `y`; otherwise returns `x`.
-#' @export
-#' @name op-null-default
+#'
 #' @examples
 #' 1 %||% 2
 #' NULL %||% 2
+#'
+#' @export
```
Comment (IndrajeetPatil) [inline R/utilities.R:16]: `#' export` should always be the last tag in the docs.

**Review by PavelBal — APPROVED** [review]

**Pull request merged**